src/pages/EmailList/index.jsx

```diff
@@ -22,10 +22,11 @@ function EmailList() {
         return newDate
     }
 
-        return (
-            <div className="border-b-2">
-                {emails.map(email => 
+    return (
```

Comment:
At this point it's worth renaming this component to be consistent with the other pages - just put Page on the end. Make sure you rename both the folder and the function